.github/workflows/ci.yml

```diff
@@ -43,7 +43,6 @@ jobs:
           - 7.4
           - 7.3
           - 7.2
-          - 7.1
```

Comment:
For the reference: Can't run PHPStan on legacy PHP 7.1 (unlike https://github.com/reactphp/promise/pull/246) because we (need to) use conditional return types which require PHPStan 1.6 which in turn requires PHP 7.2+ (https://phpstan.org/blog/phpstan-1-6-0-with-conditional-return-types). (See also https://github.com/reactphp/promise/pull/247/files#r1259622419)